src/intrinsics.ts

```diff
@@ -115,6 +115,7 @@ const ReflectiveIntrinsicObjectNames = [
     'EvalError',
     'Function',
     'Object',
+    // 'Proxy',
```

Comment:
yes... just remove the comment :)

Reply:
Ok. I'll remove the comment to expose the other issues.